packages/app/src/cli/models/extensions/specifications/app_config_privacy_compliance_webhooks.ts

```diff
@@ -0,0 +1,18 @@
+import {TransformationConfig, createConfigExtensionSpecification} from '../specification.js'
+import {AppSchema} from '../../app/app.js'
+
+const WebhookSchema = AppSchema.pick({webhooks: true}).strip()
+
+const PrivacyComplianceTransformConfig: TransformationConfig = {
+  customers_redact_url: 'webhooks.privacy_compliance.customer_deletion_url',
+  customers_data_request_url: 'webhooks.privacy_compliance.customer_data_request_url',
+  shop_redact_url: 'webhooks.privacy_compliance.shop_deletion_url',
```

Comment:
We'll finally adopt a different schema for GDPR webhooks (https://vault.shopify.io/gsd/projects/36745/decisions/4222), but we'll need to support the old one, so we can leave this for now. And we can add it the new schema later.